ui/src/views/AutogenView.vue

```diff
@@ -957,7 +957,8 @@ export default {
             if (param.name !== key) {
               continue
             }
-            if (input === undefined || input === null || input === '') {
+            if (input === undefined || input === null || (input === '' &&
```

Comment:
@Pearl1594 In my opinion it will be cleaner like this
`if (!input || (input === '' && !['updateStoragePool', 'updateHost'].includes(action.api)))` {

Reply:
@utchoang doing so would lead to the second condition not being evaluated when the input param is empty('') and the api is one among - update physicalnetwork/storagepool/host as the first condition evaluates to true - because (!input) = string not being undefined, null or empty.

Reply:
I understood! Thanks @Pearl1594